salt/metalk8s/kubeadm/init/addons/coredns.sls

```diff
@@ -0,0 +1,123 @@
+{%- from "metalk8s/map.jinja" import coredns with context %}
+
+{% set kubeconfig = "/etc/kubernetes/admin.conf" %}
+{% set context = "kubernetes-admin@kubernetes" %}
+
+include:
+  - metalk8s.salt.minion.running
+
+Create coredns ConfigMap:
+  kubernetes.configmap_present:
+    - name: coredns
+    - namespace: kube-system
+    - kubeconfig: {{ kubeconfig }}
+    - context: {{ context }} 
+    - data:
+        Corefile: |
+          .:53 {
+              errors
+              health
+              kubernetes {{ coredns.cluster_domain }} {{ coredns.reverse_cidrs }} {
+                pods insecure
+                upstream
+                fallthrough in-addr.arpa ip6.arpa
+              }
+              prometheus :9153
+              proxy . /etc/resolv.conf
+              cache 30
+              loop
+              reload
+              loadbalance
+          }
+  require:
+    - pkg: Install Python Kubernetes client
+
+Create coredns deployment:
+  kubernetes.deployment_present:
+    - name: coredns
+    - namespace: kube-system
+    - kubeconfig: {{ kubeconfig }}
+    - context: {{ context }}
+    - source: salt://metalk8s/kubeadm/init/addons/files/coredns_deployment.yaml
+  require:
+    - kubernetes: Create coredns ConfigMap
```

Comment:
If the step before depends on `Install Python Kubernetes client`, then this one (and others?) as well, no?

Reply:
hmm yes make send for all of them to depends